.github/workflows/builds.yml

```diff
@@ -2489,12 +2496,20 @@ jobs:
                 linux-headers-$(uname -r)
 
       - name: Checkout Netmap repository
+        if: steps.netmap-cache.outputs.cache-hit != 'true'
         uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
```

Comment:
`git clone` seems to work better than `actions/checkout` in https://github.com/OISF/suricata/pull/11903

But not sure it is a good path to follow

Reply:
I think the checkout action is more idiomatic